Gems/Atom/RPI/Code/Source/RPI.Public/Culling.cpp

```diff
@@ -296,13 +296,13 @@ namespace AZ
 #endif
             return worklistData;
         }
-
+            
         constexpr size_t WorkListCapacity = 5;
         using WorkListType = AZStd::fixed_vector<AzFramework::IVisibilityScene::NodeData, WorkListCapacity>;
 
 #if AZ_TRAIT_MASKED_OCCLUSION_CULLING_SUPPORTED
         static MaskedOcclusionCulling::CullingResult TestOcclusionCulling(
-                    const AZStd::shared_ptr<WorklistData>& worklistData,
+                    const AZStd::shared_ptr<WorklistData>& worklistData, 
```

Comment:
Related to the extra spaces above, you can reduce the diff of this change overall by removing the unnecessary whitespace differences

Reply:
Sorry about these. I think rebase caused them.
I will be more careful with the next PR I make.

Reply:
Sometimes there are settings in the IDE to auto-format code